e2e/cypress/integration/custom_status/custom_status_spec.js

```diff
@@ -0,0 +1,557 @@
+            // # Click on the hamburger menu in the LHS header and open the dropdown menu
+            cy.get('.sidebar-header-dropdown__icon').click();
+            cy.get('ul.dropdown-menu').should('exist');
+
+            // # Click on the "View Members" option in the dropdown menu and open the team members modal
+            cy.get('ul.dropdown-menu').findByText('View Members').click();
+            cy.get('#teamMembersModal').should('exist');
+
+            // # Search the current user's username in the search input
+            cy.get('#searchUsersInput').type(currentUser.username);
+
+            // * Custom status emoji should be visible next to the username of the current user
+            cy.get('#teamMembersModal .more-modal__row span.emoticon').should('exist').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+
+            // # Close the team members modal
+            cy.get('#teamMembersModal .close').click();
+        });
+
+        it('MM-T3850_9 should show custom status emoji next to username in the more direct messages modal', () => {
+            // # Click on the + button for Direct messages and open the Direct messages modal
+            cy.get('button[aria-label="Write a direct message"]').click();
+            cy.get('#moreDmModal').should('exist');
+
+            // # Search the current user's username in the search input
+            cy.get('#moreDmModal #react-select-2-input').type(currentUser.username);
+
+            // * Custom status emoji should be visible next to the username of the current user
+            cy.get('#moreDmModal .more-modal__row span.emoticon').should('exist').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+        });
+
+        it('MM-T3850_10 should show custom status emoji next to username in DM in LHS Direct Messages section and full custom status in channel header', () => {
+            // # Click on the search result to open the Direct messages channel
+            cy.get('#moreDmModal .more-modal__row').click();
+
+            // * Check if the channel is open and contains the channel header
+            cy.get('#channelHeaderDescription .header-status__text').should('exist');
+
+            // * Custom status text and emoji should be displayed in the channel header
+            cy.get('#channelHeaderDescription .header-status__text').should('contain', customStatus.text);
+            cy.get('#channelHeaderDescription .header-status__text span.emoticon').should('exist').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+
+            // * Custom status emoji should be visible along with username in the  LHS Direct Messages section
+            cy.get('.SidebarChannelGroup_content').contains('(you)').get('span.emoticon').should('exist').invoke('attr', 'data-emoticon').should('contain', customStatus.emoji);
+        });
+    });
+
+    describe('MM-T3852 Custom Status slash commands', () => {
```

Comment:
Please move to its own file `custom_status_6_spec.js`

```suggestion
    describe('Custom Status - Slash Commands', () => {
```